code/game/jobs/job/special_vr.dm

```diff
@@ -72,17 +72,17 @@
 	departments = list(DEPARTMENT_CIVILIAN)
 	department_flag = CIVILIAN
 	faction = "Station"
-	total_positions = -1
-	spawn_positions = -1
+	total_positions = 1
+	spawn_positions = 1
 	supervisors = "the spirit of laughter"
 	selection_color = "#515151"
 	economic_modifier = 1
-	access = list()
-	minimal_access = list()
+	access = list(access_entertainment)
+	minimal_access = list(access_entertainment)
 	job_description = "A Clown is there to entertain the crew and keep high morale using various harmless pranks and ridiculous jokes!"
 	alt_titles = list("Clown" = /datum/alt_title/clown, "Comedian" = /datum/alt_title/comedian, "Jester" = /datum/alt_title/jester)
 	whitelist_only = 1
-	latejoin_only = 1
+	latejoin_only = 0
```

Comment:
Won't making it not latejoin only make it show up in the list of occupations even as it is whitelisted? Or is it being whitelisted prevents that?

Reply:
The idea here is that making it not latejoin only, and giving them a proper spawn, will yes make it show up on the job list, but it should be crossed out and say "whitelisted" or something to that effect. If it is not, then this can be reverted. I'll check and confer.

Reply:
Even if it will, I feel like this is a bit unnecesary flood to job list as a whole for vast majority of playerbase. Plus, considering nature of the jobs, I don't think roundstart spawnin is that relevant at all (and can only think of less than handful of jobs in general for which it is).

Reply:
I've reverted this and taken away the clown and mime roundstart spawns. I'll bring the question of what we want to do with clowns and mimes to a staff discussion instead.